passageidentity.gemspec

```
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name = 'passageidentity'
-  s.version = '0.0.8'
+  s.version = '1.0.0'
```

any reason why we jumped to 1.0.0 here?

---

I was doing it to indicate a breaking change, but in [the SemVer spec](https://semver.org/#spec-item-4) it says:
> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

So this can just be
```suggestion
  s.version = '0.1.0'
```